File: P2P Task/netsalary.py
basicSalary = int(input("enter your basic salary"))
pf = 0
da = 0
hra = 0
nightAllowance = 500
if basicSalary < 10000:
    pf = 0.03 * basicSalary
    da = 0.04 * basicSalary
    hra = 0.025 * basicSalary
elif basicSalary >= 10001 and basicSalary <= 25000:
    pf = 0.05 * basicSalary
    da = 0.06 * basicSalary
    hra = 0.045 * basicSalary
elif basicSalary > 25000:
    pf = 0.07 * basicSalary
    da = 0.05 * basicSalary
    hra = 0.06 * basicSalary

netSalary = basicSalary + da + hra + pf + nightAllowance
grossSalary = netSalary-pf
print("Net salary : ", netSalary)
print(grossSalary)

File: P2P Task/netSalary.js

function calculateNetSalary() {

    let salary = parseFloat(document.getElementById("basicSalary").value);


    let nightShift = document.getElementById("nightShift").checked;
    let nightShiftAllowance = 0;
    if (nightShift) {
        nightShiftAllowance = 500;
    }

    let pf = 0;
    let da = 0;
    let hra = 0;
    if (salary < 10000) {
        pf = 0.03 * basicSalary;
        da = 0.04 * basicSalary;
        hra = 0.025 * basicSalary;
    } else if (salary >= 10001 && salary <= 25000) {
        pf = 0.05 * salary;
        da = 0.06 * salary;
        hra = 0.045 * salary;
    } else if (salary > 25000) {
        pf = 0.07 * salary;
        da = 0.05 * salary;
        hra = 0.06 * salary;
    }


    let netSalary = salary + da + hra + nightShiftAllowance

    document.getElementById("result").value = netSalary;




}

